src/eq_schema/Question.js

```diff
@@ -27,7 +27,24 @@ class Question {
   }
 
   buildAnswers(answers) {
-    return answers.map(answer => new Answer(answer));
+    const answerArray = flatten(
+      answers.map(answer => {
+        if (answer.hasOwnProperty("other")) {
+          const childAnswer = {
+            id: answer.other.answer.id,
+            alias: answer.other.answer.id,
+            mandatory: answer.other.answer.mandatory,
+            type: answer.other.answer.type,
+            label: answer.other.answer.label,
+            description: answer.other.answer.description,
+            parentAnswerId: answer.id
```

Comment:
The fact that we're accessing properties of answer here suggests to me that building the other Answer should live within the Answer class. See [tell don't ask](https://martinfowler.com/bliki/TellDontAsk.html).

Could we re-use the constructor? e.g. `new Answer(answer.other.answer);` or if not, would it be cleaner to introduce some kind of static method e.g. `Answer.buildChildAnswer()`?

Reply:
Good point, I've added in a static method for the child answer and it does look a lot cleaner.